Web_Development/LightFlow/lightflow/cli/main.py

```
@@ -0,0 +1,326 @@
+        workflow = loader.load_workflow(workflow_file)
+        
+        # Build DAG
+        dag_builder = DAGBuilder()
+        dag = dag_builder.build_dag(workflow)
+        
+        # Generate visualization
+        if output:
+            output_file = f"{output}.{format}"
+            dot_source = dag_builder.visualize_dag(dag, output_file)
+        else:
+            dot_source = dag_builder.visualize_dag(dag)
```

Ruff (0.12.2)

Local variable `dot_source` is assigned to but never used

Remove assignment to unused variable `dot_source`

(F841)